src/models/apiModels.ts

```diff
@@ -87,6 +88,12 @@ export interface Event {
   confirmed?: boolean;
 }
 
+export interface EventOptions {
```

Comment:
Maybe we can remove this and just use the `JoinEventPayload` instead if we decide to only use on state where I commented. I think then it would also make sense to rename `JoinEventPayload` to `EventOptions` IMO, but renaming that is definitely for another PR and not something🙌